_inc/client/traffic/private.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { FEATURE_PRIVATE_JETPACK } from 'lib/plans/constants';
+import { ModuleSettingsForm as moduleSettingsForm } from 'components/module-settings/module-settings-form';
+import { ModuleToggle } from 'components/module-toggle';
+import SettingsCard from 'components/settings-card';
+import SettingsGroup from 'components/settings-group';
+import { FormFieldset } from 'components/forms';
+
+class Private extends React.Component {
+	render() {
+		const module_enabled = this.props.getOptionValue( 'private' );
+		return (
+			<SettingsCard
+				{ ...this.props }
+				module="private"
+				feature={ FEATURE_PRIVATE_JETPACK }
+				hideButton
+			>
+				<SettingsGroup
+					hasChild
+					module={ { module: 'private' } }
+					support={ {
+						text: __( 'Jetpack private sites' ),
```

Comment:
I don't think we need to specify "Jetpack" here.

Reply:
Removed 👍